src/commands/install-node.yml

```diff
@@ -59,11 +59,27 @@ steps:
         - run:
             name: Install and use node version
             command: |
-              # if("<<parameters.version>>" -eq "latest") {
-              #   choco install nodejs
-              # } else {
-              #   $r = Invoke-WebRequest https://nodejs.org/download/release/index.json | ConvertFrom-Json
-              #   $latest_of_version = ($r | where { $_.version -like "v<<parameters.version>>*" })[0].version.substring(1)
-              #   choco install nodejs --version $latest_of_version --allow-downgrade
-              # }
+              $nodeIndex = "https://nodejs.org/download/release/index.json"
+
+              if ("<<parameters.version>>" -eq "latest") {
+                choco install nodejs
+              } elseif ("<<parameters.version>>" -eq "lts") {
+                $registry =  (Invoke-WebRequest $nodeIndex).content
+                $lts = $registry |
+                    jq -c -r 'first(.[] | select(.lts != false)).version | ltrimstr(\"v\")'
+
+                choco install nodejs --version $lts --allow-downgrade
+              } elseif ("<<parameters.version>>" -eq "maintenance") {
+                $registry =  (Invoke-WebRequest $nodeIndex).content
+
+                $active = $registry | jq -c -r 'first(.[] | select(.lts != false)).lts'
+                $maintenance = $registry |
+                    jq -c -r --arg active $active `
+                    'first(.[] | select(.lts != $active and .lts != false)).version | ltrimstr(\"v\")'
```

Comment:
To find the maintenance LTS version:

1. Get the current active LTS from the registry
```powershell
$active = $registry | jq -c -r 'first(.[] | select(.lts != false)).lts'
```
2. Iterate over the registry to find the first object in which `lts != current LTS` and `lts != false`:
```powershell
jq -c -r --arg active $active `
  'first(.[] | select(.lts != $active and .lts != false)).version | ltrimstr(\"v\")'
```

node.js releases: https://nodejs.org/en/about/releases/

Reply:
Very nice! 😁 
![image](https://user-images.githubusercontent.com/1084688/134971711-31be843c-d588-4056-ae65-06a277ab9729.png)